apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/LoadedFilterManager.java

```diff
@@ -37,7 +39,7 @@ public class LoadedFilterManager {
   private EntityManager entityManager;
 
   /** The filter set. */
-  private List<LoadedFileFilter> filters;
+  @Getter private List<LoadedFileFilter> filters;
```

Comment:
@rcrupp  I've used lombok before but I'm not familiar with seeing an @ Getter get used like this. What's the reasoning?

Reply:
The annotation can be applied at either the class or method level.  At the class level it exposes every field with a getter.  At the field level it adds a getter for just that field.  In this case the class only had a getter for `filters` so we don't want to accidentally add one for `entityManager`.  Adding the annotation to `filters` directly preserves the existing encapsulation of `entityManager`.

Reply:
Oh!! Thank you for explaining @brianburton !